test/to_vec.jl

```diff
@@ -83,6 +83,18 @@ end
                 test_to_vec(Dict(:a=>3 + 2im, :b=>randn(T, 10, 11), :c=>(5+im, 2-im, 1+im)))
             end
         end
+
+        @testset "Generator(identity, x)" begin
```

Comment:
Would be good to include at least one test with an array that doesn't contain `Real`s.